crates/test/src/lib.rs

```diff
@@ -20,6 +20,31 @@ macro_rules! console_log {
     )
 }
 
+/// A macro used to configured how this test is executed by the
+/// `wasm-bindgen-test-runner` harness.
+///
+/// This macro is invoked as:
+///
+///     wasm_bindgen_test_configure!(foo bar baz);
+///
+/// where all of `foo`, `bar`, and `baz`, would be recognized options to this
+/// macro. The currently known options to this macro are:
+///
+/// * `run_in_browser` - requires that this test is run in a browser rather than
+///   node.js, which is the default for executing tests.
+///
+/// This macro may be invoked at most one time per test suite.
```

Comment:
A "test suite" is a test crate, right? like `tests/browser.rs` or `tests/all/*`? *Not* per-module, right? Would be good to clarify this point.

Reply:
True!